JS/Lesson32/onder/my-app/src/components/RecipeDetail/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const RecipeDetail = ({ recipe, onClearSelection }) => {
```

Comment:
Must have: export the RecipeDetail component
```suggestion
export const RecipeDetail = ({ recipe, onClearSelection }) => {
```